qTox build failed

```
In file included from ../src/platform/statusnotifier/enums.h:25:0,
                 from ../src/platform/statusnotifier/enums.c:24:
../src/platform/statusnotifier/statusnotifier.h:29:35: fatal error: gdk-pixbuf/gdk-pixbuf.h: No such file or directory
compilation terminated.
Makefile:2513: recipe for target 'enums.o' failed
make: *** [enums.o] Error 1
```
Arch linux current
build in chroot ([extra-x86_64-build](https://wiki.archlinux.org/index.php/DeveloperWiki:Building_in_a_Clean_Chroot))

--- a/qtox-git/PKGBUILD
+++ b/qtox-git/PKGBUILD
@@ -3,7 +3,7 @@
 
 _pkgname=qtox
 pkgname=qtox-git
-pkgver=r3026.59d5e04
+pkgver=r4114.f61332b
 pkgrel=1
 pkgdesc='Powerful Tox client written in C++/Qt that follows the Tox design guidelines.'
 arch=('i686' 'x86_64' 'arm' 'armv6h' 'armv7h')
@@ -19,6 +19,8 @@ depends=(
   'qt5-svg'
   'qrencode'
   'toxcore'
+  'sqlcipher'
+  'gdk-pixbuf2'
 )
 makedepends=('git' 'qt5-tools')
 provides=("$_pkgname")

